htdocs/societe/class/companybankaccount.class.php

```diff
@@ -193,17 +187,6 @@ class CompanyBankAccount extends Account
 
 	public $country_code;
 
-
-	/**
-	 * @var string owner
-	 */
-	public $proprio;
```

Comment:
To have transition version, we are keeping $this->proprio assignation, for example line 520.
Don't we must keep also the declaration here and add the flag
`@deprecated use $owner`

Reply:
$this->proprio is in account.class.php and is set to deprecated

Reply:
https://github.com/Dolibarr/dolibarr/blob/2e938fe9ef32a464aeb0c7f782ec1d261da7f0f5/htdocs/compta/bank/class/account.class.php#L397-L406

Reply:
if you try to access proprio (private) it will return owner_name